stats/losses/core.py

```diff
@@ -10,6 +10,35 @@ def __init__(self, api_key):
         self.api_key = api_key
         print("created Config")  
 
+class StatsService():
+
+    def __init__(self):
+        API_KEY = os.getenv('RIOT_API_KEY_APP')
+        self.riotProxy = RiotProxy(API_KEY)
+
+    def calculate_dps_threats(self, names):
+        res = []
+
+        for name in names:
+            dps_threat = self.calculate_dps_threat(name)
+            res.append(dps_threat)
+
+        return res
+
+    def calculate_dps_threat(self, name):
+
+        postgresProxy = PostgresProxy()
```

Comment:
you should make PostgersProxy with'able

```
with PostgresProxy() as f:
    # write the code
```

That way you don't have to worry about calling close, mostly in the case that any code between open and close throws an exception

Reply:
Oh i forgot all about that! ty ty